components/Hero/StoryHero.tsx

```diff
@@ -111,7 +110,7 @@ export const StoryHero = ({
         <div className={styles.content(!!filename, isVerticalHero, isLeftImage)}>
           <div className={cnb(
             styles.tabSection(!!tabColorValue, isVerticalHero, isLeftImage),
-            accentBorderColors[paletteAccentColors[tabColorValue || '']],
+            accentBorderColors[paletteAccentColors[tabColorValue] as AccentBorderColorType],
```

Comment:
Instead of typecasting it's probably better practice to add a type guard. Ie: check for valid values.